File: frontend/StudentManagement.Shared/Dtos/BaoCao/BaoCaoMonResponse.cs
namespace StudentManagement.Shared.Dtos.BaoCao;

public class BaoCaoMonResponse
{
    public string IdLop { get; set; } = string.Empty;
    public string TenLop { get; set; } = string.Empty;
    public string IdMonHoc { get; set; } = string.Empty;
    public string TenMonHoc { get; set; } = string.Empty;
    public string IdHocKy { get; set; } = string.Empty;
    public int TongHocSinh { get; set; }
    public int SoLuongDat { get; set; }
    public double TyLeDat { get; set; }
}

File: backend/src/main/java/com/demo/student_management/dto/baocao/BaoCaoMonResponse.java
package com.demo.student_management.dto.baocao;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class BaoCaoMonResponse {

    private String idLop;
    private String tenLop;

    private String idMonHoc;
    private String tenMonHoc;

    private String idHocKy;

    private Integer tongHocSinh;
    private Integer soLuongDat;

    private Double tyLeDat;
}